src/peepsClient.js

```diff
@@ -0,0 +1,50 @@
+class PeepsClient {
+  async loadPeeps() {
+    try {
+      const response = await fetch('https://chitter-backend-api-v2.herokuapp.com/peeps');
+      const data = await response.json();
+  
+      return data.map(peep => ({
```

Comment:
Well done on formatting the API response object in client before it goes into the model array, I struggled with this in my own version. The complexity of your formatted object is quite impressive.